applications/concept-cat/src/main/java/no/ccat/controller/ConceptSearchESQueryBuilder.java

```diff
@@ -51,4 +51,20 @@ ConceptSearchESQueryBuilder addParams(Map<String, String> params) {
         }
         return this;
     }
+
+    ConceptSearchESQueryBuilder boostPrefLabel(String preflabel) {
+        if (preflabel == null || preflabel.isEmpty()) {
+            return this;
+        }
+        QueryBuilder nbQuery = QueryBuilders.matchPhrasePrefixQuery("prefLabel.nb", preflabel).analyzer("norwegian").maxExpansions(15);
+        QueryBuilder noQuery = QueryBuilders.matchPhrasePrefixQuery("prefLabel.no", preflabel).analyzer("norwegian").maxExpansions(15);
+        QueryBuilder nnQuery = QueryBuilders.matchPhrasePrefixQuery("prefLabel.nn", preflabel).analyzer("norwegian").maxExpansions(15);
+        QueryBuilder enQuery = QueryBuilders.matchPhrasePrefixQuery("prefLabel.en", preflabel).analyzer("english").maxExpansions(15);
+
+        composedQuery.should(nbQuery.boost(2));
+        composedQuery.should(noQuery.boost(2));
+        composedQuery.should(nnQuery.boost(2));
+        composedQuery.should(enQuery.boost(2));
```

Comment:
i stedet af boost, vi kan fikse q parameter implementasjon i
no.ccat.controller.ConceptSearchParamHandlers

        QueryBuilder fuzzyQuery= QueryBuilders.simpleQueryStringQuery(searchText);
        QueryBuilder prefLabelQuery = ConceptSearchParamHandlers.prefLabel(value, queryBuilder);
      
        return QueryBuilders.boolQuery().should(fuzzyQuery).should(prefLabelQuery);

Reply:
Jeg testet dette, og det hadde ikke den ønskede effekt på rekkefølgen av resultater så jeg beholder det jeg har.